spring/src/main/java/org/axonframework/spring/config/annotation/SpringBeanParameterResolverFactory.java

```diff
@@ -31,12 +34,14 @@
 import java.lang.reflect.Executable;
 import java.lang.reflect.Parameter;
 import java.util.Map;
+import java.util.Optional;
 
 /**
  * ParameterResolverFactory implementation that resolves parameters in the Spring Application Context. A parameter can
  * be resolved as a Spring bean if there is exactly one bean assignable to the parameter type. If multiple beans are
- * available and one is marked as primary, that bean is chosen. Note that when multiple beans are marked as primary,
- * either one can be selected as parameter value.
+ * available the desired one can be designated with a {@link Qualifier} annotation on the parameter. By absence of a
```

Comment:
As you've pointed out in the issue for this ticket, just saying 'Qualifier' might let people think it's the `javax.inject.Qualifier`. So my suggestion is to add the full import statement to this link, like so:
`{@link org.springframework.beans.factory.annotation.Qualifier}`